### ISSUE TITLE
Custom build - Adding JAR directly to project

### ISSUE THREAD

**Issue by sambengtson** [OneSignal/OneSignal-Android-SDK#535]
**Description:**
Due to a gradle build conflict with play-services-location,  I decided to set the play-services-location verison to 15.0.1 of the SDK and rebuild.  I then included the output jar directly into my project.  I modified the example code to call OneSignal.init directly with my app id hard-coded

`OneSignal.init(activity, "REMOTE", "<valid-id>");`

I am receiving the following errors: 

> OneSignal: Unknown error getting GCM Token

and

> No such service ComponentInfo{com.tsm.chip.probation/com.onesignal.SyncJobService

I realize this isn't the intended way to include OneSignal in a project.  Any tips to get this going?

**Environment**
Android 8.0

**Steps to Reproduce Issue:**
Clone the project
Change the play-services-location version to 15.0.1
Build with gradle
Add the output JAR directly to another project

**Comment by jkasten2** [comment]
@sambengtson Using the OneSignal jar requires adding a bunch of `AndroidManifest.xml` settings which the normal gradle setup adds for you. This can be found below.
https://documentation.onesignal.com/docs/android-sdk-setup-eclipse#section-2-androidmanifest-xml

It is best to use our normal gradle setup guide as it is easier to maintain in a number of ways. Could provide the exact build error you were with the normal setup? As well as your root `build.gradle` and your `app/build.gradle`.

Thanks.

**Comment by sambengtson** [comment]
The errors are a little hard to follow since it is actually a cordova project.  I had to make a plugin to handle notifications in native code.  However,  I was able to make gradle work.  I think the problem was mostly due to my lack of knowledge with it.
